src/views/templates/index.jade

```diff
@@ -0,0 +1,69 @@
+-
+    actorsAndMovies = JSON.parse(actorsAndMovies);
+    var movies = [];
+    var actors = actorsAndMovies.actors;
+    for (var i = 0; i < actorsAndMovies.movies.length; i++) {
+        var movie = actorsAndMovies.movies[i];
+        for (var j = 0; j < movie.relationships.length; j++) {
+            var relationship = movie.relationships[j];
+            var actor1 = actors[relationship[0]];
+            var age1 = movie.year - new Date(actor1.born).getFullYear();
+            var actor2 = actors[relationship[1]];
+            var age2 = movie.year - new Date(actor2.born).getFullYear();
+            var newMovie = {
+                name: movie.name,
+                difference: Math.abs(age1 - age2),
+                relationship: [
+                    {
+                        age: age1,
+                        gender: actor1.gender,
+                        name: actor1.name
+                    },
+                    {
+                        age: age2,
+                        gender: actor2.gender,
+                        name: actor2.name
+                    }
+                ],
+                year: movie.year
+            };
+            movies.push(newMovie);
+        }
+    }
+    movies.sort(function (a, b) {
+        if (a.difference > b.difference) {
+            return -1;
+        } else if (a.difference < b.difference) {
+            return 1;
+        } else {
+            return 0;
+        }
+    });
```

Comment:
It's pretty nasty having this big chunk of javascript in the jade template; I'm looking into ways for this to not be a thing.